build_from.py

```diff
@@ -14,31 +17,21 @@
 from regparser.diff import treediff
 from regparser.builder import Builder, LayerCacheAggregator
 
+pyparsing.ParserElement.enablePackrat()
```

Comment:
Noted. I only enabled it to see if it would speed things up appreciably; it did not.